src/Microsoft.DotNet.Darc/src/DarcLib/Helpers/FeedConstants.cs

```diff
@@ -15,16 +15,16 @@ public class FeedConstants
         {
             // Matches package feeds like
             // https://dnceng.pkgs.visualstudio.com/public/_packaging/darc-pub-arcade-fd8184c3fcde81eb27ca4c061c6e171f418d753f-1/nuget/v3/index.json
-            @"https://(?<organization>\w+).pkgs.visualstudio.com/(public/){0,1}_packaging/darc-(?<type>(int|pub))-(?<repository>.+?)-(?<sha>[A-Fa-f0-9]{7,40})-?(?<subversion>\d*)/nuget/v\d+/index.json",
+            @"https://(?<organization>\w+).pkgs.visualstudio.com/(public/){0,1}_packaging/" + MaestroManagedFeedNamePattern + @"/nuget/v\d+/index.json",
             // Matches package feeds like
             // https://pkgs.dev.azure.com/dnceng/public/_packaging/darc-pub-dotnet-wpf-8182abc8/nuget/v3/index.json
-            @"https://pkgs.dev.azure.com/(?<organization>\w+)/(public/){0,1}_packaging/darc-(?<type>(int|pub))-(?<repository>.+?)-(?<sha>[A-Fa-f0-9]{7,40})-?(?<subversion>\d*)/nuget/v\d+/index.json"
+            @"https://pkgs.dev.azure.com/(?<organization>\w+)/(public/){0,1}_packaging/"+ MaestroManagedFeedNamePattern + @"/nuget/v\d+/index.json"
```

Comment:
I think you can actually use `@$` here.

Reply:
It ends up conflicting with the {0,1} in the regex.